HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -7339,7 +7339,7 @@ Patch "Disable Spin Charge Camera" in "Camera" by "Hyper" does "Disables the zoo
     );
 }
 
-Patch "Disable Spin Slash on Drop Dash" in "Gameplay" by "Hyper"
+Patch "Disable Spin Slash on Drop Dash" in "Gameplay" by "Hyper" does "Disables performing the Spin Slash when getting close to an enemy while drop dashing."
```

Comment:
Change to "Disables performing a Spin Slash after a Drop Dash whilst in combat."